.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
I'm kind of new to GitHub and pull requests. Would it be OK if I just remove it and make another commit?